.github/workflows/ci.yml

```diff
@@ -34,11 +34,13 @@ jobs:
       - run: exit 0
 
   lint:
-    runs-on: ubuntu-24.04
+    runs-on: macos-15 # latest
     steps:
-      - uses: actions/checkout@v4
-      - name: GitHub Action for SwiftLint
-        uses: norio-nomura/action-swiftlint@3.2.1
+      - uses: actions/checkout@v3
+      - name: Run Swift-Format
+        run: |
+          brew install swift-format
+          swift-format lint --recursive --strict .
```

Comment:
Can we specify a swift-format version so that we could use the same version across the team.

Reply:
Great question but unfortunately for swift-format, I don't think there is an easy way to specify a version. The only option I can think of is building from source, which will be a pain. We can do something like brew install swift-format@version, but as new versions will be released, brew won't keep the old versions.

Using the latest from brew should work fine as we can just do brew upgrade whenever new swift-format versions are released with any breaking changes.

Reply:
The problem with this is that we will need to do a reformatting PR as soon as new versions of Swift-format are released with incompatible changes but I'm not sure how often that would be.

Reply:
@xiazhvera Any concerns with this approach or are we good to ship it?

Reply:
No, good to go.